src/pkg/rbac/model/model.go

```diff
@@ -0,0 +1,50 @@
+package model
+
+import (
+	"github.com/astaxie/beego/orm"
+	"time"
+)
+
+func init() {
+	orm.RegisterModel(&RolePermission{})
+	orm.RegisterModel(&RbacPolicy{})
+}
+
+// RolePermission records the relations of role and permission
+type RolePermission struct {
+	ID           int64     `orm:"pk;auto;column(id)"`
+	RoleType     string    `orm:"column(role_type)"`
+	RoleID       int64     `orm:"column(role_id)"`
+	RBACPolicyID int64     `orm:"column(rbac_policy_id)"`
+	CreationTime time.Time `orm:"column(creation_time);auto_now_add" json:"creation_time"`
+}
+
+// TableName for role permission
+func (rp *RolePermission) TableName() string {
+	return "role_permission"
+}
+
+// RbacPolicy records the policy of rbac
+type RbacPolicy struct {
```

Comment:
Can `RbacPolicy` be `PermissionPolicy` because there is a type named `Policy` in permission pkg which also includes `Resource`, `Action` and `Effect`?

Reply:
mapping corresponding the table of rbac_policy.